tests/e2e/tailscale/tailscale_test.go

```diff
@@ -14,7 +14,7 @@ import (
 // Valid nodeOS: generic/ubuntu2310, opensuse/Leap-15.3.x86_64
 var nodeOS = flag.String("nodeOS", "generic/ubuntu2310", "VM operating system")
 var serverCount = flag.Int("serverCount", 1, "number of server nodes")
-var agentCount = flag.Int("agentCount", 1, "number of agent nodes")
+var agentCount = flag.Int("agentCount", 2, "number of agent nodes")
 var ci = flag.Bool("ci", false, "running on CI")
 var local = flag.Bool("local", false, "deploy a locally built K3s binary")
 
```

Comment:
We would need to add some extra logic to verify that the number of nodes in the cluster is serverCount + agentCount to make sure that both agents joined correctly. E.g. after this line https://github.com/k3s-io/k3s/blob/master/tests/e2e/tailscale/tailscale_test.go#L84 we could check if the len(nodes) is equal to "serverCount + agentCount"